docs/osindex.html

```diff
@@ -42,6 +42,7 @@
     <div class='map-overlay-inner'>
         <button id='request'>Find localBuildings</button>
         <div>Total features: <span id='feature-count'></span></div>
+        <div>Total area <b>(metres squared)</b>: <span id='area'></span></div>
```

Comment:
typo: https://en.wikipedia.org/wiki/Square_metre

Reply:
Added a sup tag to the 2 so it reads m^2